### PR TITLE
Hex like deref

### PR DESCRIPTION
**Pull request by Grazfather**
I didn't like the way that hexdump showed offsets. I make it more obvious and colourful.

Old:
1__tmux

New:
1__tmux

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -7616,7 +7618,8 @@ def _hexdump(self, start_addr, length, arrange_as, offset=0):
             sym = "<{:s}+{:04x}>".format(*sym) if sym else ''
             mem = read_memory(cur_addr, l)
             val = struct.unpack(fmt_pack, mem)[0]
-            lines.append(fmt_str % (cur_addr, (i + offset) * l,  sym, val))
+            lines.append(fmt_str.format(base=Color.colorify("{:#x}".format(cur_addr), attrs=base_address_color),
+                                        offset=i * l, sym=sym, val=val))
```
Comment (wbowling) [inline gef.py]: This is missing the offset from the last PR:

`offset=(i + offset) * l`

**Comment by wbowling** [comment]
Looking fancy :)

File: gef.py (hunk 1 above)
Comment (Grazfather) [reply gef.py]: Oh good eye, I had some commits mixed up when I rebased.

**Comment by wbowling** [comment]
LGTM

**Comment by Grazfather** [comment]
Actually, if I use `format_address` it's even more consistent. What do you think? (it's busier on 64 bit)

1__tmux

**Comment by wbowling** [comment]
It’s amazing what a bit of colour can do :) extra padding brings it all in line.

LGTM

**Pull request merged**